server/models.ts

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable global-require */
+import path from 'path';
+import Sequelize from 'sequelize';
+
+if (process.env.NODE_ENV !== 'production' && process.env.NODE_ENV !== 'test') {
+	require(path.join(process.cwd(), 'config.js'));
+}
+
+// @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
+const useSSL = process.env.DATABASE_URL.indexOf('localhost') === -1;
+// @ts-expect-error ts-migrate(2351) FIXME: This expression is not constructable.
+export const sequelize = new Sequelize(process.env.DATABASE_URL, {
+	logging: false,
+	dialectOptions: { ssl: useSSL ? { rejectUnauthorized: false } : false },
+	pool: {
+		max: process.env.SEQUELIZE_MAX_CONNECTIONS
+			? parseInt(process.env.SEQUELIZE_MAX_CONNECTIONS, 10)
+			: 5, // Some migrations require this number to be 150
+		// idle: 20000,
+		// acquire: 20000,
+	},
+});
+
+/* Change to true to update the model in the database. */
+/* NOTE: This being set to true will erase your data. */
+if (process.env.NODE_ENV !== 'test') {
+	sequelize.sync({ force: false });
+}
+
+/* Create standard id type for our database */
+sequelize.idType = {
+	primaryKey: true,
+	type: Sequelize.UUID,
+	defaultValue: Sequelize.UUIDV4,
+};
+
+/* Import and create all models. */
+/* Also export them to make them available to other modules */
+export const Branch = sequelize.import('./branch/model');
```

Comment:
The main change in this file is that the `.js` extensions were stripped from the `sequlize.import` paths.